monitoring_versioned_sidebars/22.10-sidebar.yaml

```diff
@@ -164,6 +164,10 @@ sidebar:
       - prerequisites-works
       - managing-ncb
       - visualize-ncb
+  - id: i-vertix-vault
```

Comment:
Non metterei il Vault come menu principale, lo metterei nella sezione del Host Discovery

Reply:
Volevo sottolineare la sua nuova funzione mettendolo in evidenza.
Casomai ne parliamo.